Gemfile.lock

```diff
@@ -624,7 +626,7 @@ GEM
     zeitwerk (2.6.0)
 
 PLATFORMS
-  ruby
+  x86_64-linux
```

Comment:
Will this (and the `google-protobuf` change on L264) cause any problems for people developing on other systems? (are any of us developing outside of a Linux environment anyway? maybe this doesn't matter)

Reply:
Good point about google-protobuf.  Interesting approach to speeding up the testing.  I am all for faster testing too.  These took close to an hour when I ran them serially.